app/views/lists/show.html.erb

```diff
@@ -16,6 +15,7 @@
         <% if task.persisted? %>
           <tr>
             <td><%= task.title %></td>
+            <td><%= task.status ? 'Done' : 'To Do' %></td>
```

Comment:
Try to keep `views` slim and don't put much logic in them. For this case you should have helper method `task_status` that would contain logic for what should be displayed and what not.

See https://www.rubyguides.com/2020/01/rails-helpers/